components/alert/alert.html

```diff
@@ -27,20 +27,20 @@ <h1 class="home-header-title"><a href="/">WINGS UI</a></h1>
       <aside class="doc-sidebar">
         <ul class="sidebar-list ls-none mp-m">
           <strong class="sidebar-header font-m">Components</strong>
```

Comment:
`ul` should have `li` as direct child, wrap strong inside `li`

Reply:
this is like the heading of the list. we can do this atleast i think.

Reply:
Probably we can move it outside of ul then

Reply:
i'm not sure of doing it now. will do after all components are done.